acceptance/tests/environment/can_enumerate_environments.rb

```diff
@@ -56,7 +56,7 @@ def curl_master_from(agent, path, headers = '', &block)
 with_puppet_running_on(master, master_opts) do
   agents.each do |agent|
     step "Ensure that an unauthenticated client cannot access the environments list" do
-      on agent, "curl --tlsv1 -ksv https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
+      on agent, "curl --tlsv1 -s -v https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
```

Comment:
Since the test is testing that only authenticated agents can enumerate environments, we could just run the curl command on the master, and eliminate the issues with AIX and OSX. That way we know we're on a sane system that supports `curl -k` with and without client certs.